### DISCUSSION TITLE
OneNord theme port to Intelijj?

### DISCUSSION THREAD

**Discussion by acunidev** [rmehri01/onenord.nvim#56] [Q&A] [answered]
Hi! I really love your theme, being using it in Neovim for a while.
Lately I'm using IntelliJ IDEA too and there isn't any theme like this one.
Would you consider perhaps porting this theme to IntelliJ too?
Thanks :)

**Comment by rmehri01** [comment, marked as answer]
Hey, thanks I'm glad you like the theme! I'll consider it since I occasionally use JetBrains editors too but no promises depending on how easy it is to make and maintain 😃

**Reply by acunidev** [reply]
Thanks for the reply!
That would be awesome! 😀
Hopefully is not hard hahaha

**Reply by rmehri01** [reply]
I think it would be fun to try Kotlin anyways so I'll give it a go after my finals and let you know!

**Reply by rmehri01** [reply]
Hey @acunidev, I just released a first version: https://github.com/rmehri01/onenord-jetbrains. It should be on the marketplace so I'd appreciate if you could try it out and let me know if you run into any issues!

**Reply by acunidev** [reply]
Hi! I just download it, looks really nice 😀
Sure! If I run into any issues, I will let you know 
Thanks again  😀

**Reply by rmehri01** [reply]
No problem, glad you like it! 😄
